src/layout/Skill.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useCallback } from "react"
+import { useDispatch } from "react-redux"
+import {
+  putSkill,
+  getSkills,
+  deleteSkill,
+} from "../features/skills/skillsSlice"
+import { StyledInput } from "../elements/Input"
+import { ReactComponent as UpdateIcon } from "../assets/icons/update-icon.svg"
+import { ReactComponent as EditIcon } from "../assets/icons/edit-icon.svg"
+import { ReactComponent as RemoveIcon } from "../assets/icons/remove-icon.svg"
+
+function Skill({ skill }) {
+  const dispatch = useDispatch()
+  const [inputValue, setInputValue] = useState()
```

Comment:
Let's have more specific naming to the state for skill name, so if we'll have more inputs in this component -> we won't need to rename this one.